sdcm/cluster.py

```diff
@@ -2747,7 +2747,7 @@ def _gen_cqlsh_cmd(self, command, keyspace, timeout, connect_timeout):
         return f'{cqlsh_cmd} {options} -e {command} {host}'
 
     def run_cqlsh(self, cmd, keyspace=None, timeout=120, verbose=True, split=False, connect_timeout=60,
-                  num_retry_on_failure=1):
+                  num_retry_on_failure=1, retry_interval=3):
```

Comment:
Not sure I really like the internal retry in this function

And that this default 3s fits all the usages that have retires

Reply:
The only current usage is in the `set_system_auth_rf` with 3 retries. Don't see any problem there.
Plus new addition in this fixed nemesis.

Reply:
o.k. doesn't warrants a change,

just bug me in the eye that it's not align to other apis we are using `@retrying` or it's friends